model/project_parser.go

```diff
@@ -567,14 +566,19 @@ func GetProjectFromBSON(data []byte) (*Project, error) {
 // and sets the project's identifier field to identifier. Tags are evaluated. Returns the intermediate step.
 // If reading from a version config, LoadProjectForVersion should be used to persist the resulting parser project.
 // opts is used to look up files on github if the main parser project has an Include.
-func LoadProjectInto(ctx context.Context, data []byte, opts GetProjectOpts, identifier string, project *Project) (*ParserProject, error) {
+func LoadProjectInto(ctx context.Context, data []byte, opts *GetProjectOpts, identifier string, project *Project) (*ParserProject, error) {
 	intermediateProject, err := createIntermediateProject(data)
 	if err != nil {
 		return nil, errors.Wrapf(err, LoadProjectError)
 	}
 
 	// return intermediateProject even if we run into issues to show merge progress
 	for _, path := range intermediateProject.Include {
+		if opts == nil {
+			grip.Critical(message.Fields{
+				"message": "Trying to open include files with empty opts",
```

Comment:
Can we do message.WithError(errors.WithStack(<include an error here>)) ? 
Without the stack trace, we really don't have any information about how to fix this.

Reply:
We also want to break out of the loop if we hit this.

Reply:
message.WithError didn't exist and the WithStack wanted an error so this is what i came up with.